packages/thirdweb/src/exports/extensions/marketplace.ts

```diff
@@ -31,6 +31,16 @@ export {
   type CurrencyPriceForListingParams,
 } from "../../extensions/marketplace/direct-listings/read/currencyPriceForListing.js";
 
+// isDirectListingSupported
+export { isCreateListingSupported } from "../../extensions/marketplace/__generated__/IDirectListings/write/createListing.js";
+export { isCancelListingSupported } from "../../extensions/marketplace/__generated__/IDirectListings/write/cancelListing.js";
+export { isBuyFromListingSupported } from "../../extensions/marketplace/__generated__/IDirectListings/write/buyFromListing.js";
+
+// isEnglishAuctionSupported
+export { isCreateAuctionSupported } from "../../extensions/marketplace/__generated__/IEnglishAuctions/write/createAuction.js";
+export { isBidInAuctionSupported } from "../../extensions/marketplace/__generated__/IEnglishAuctions/write/bidInAuction.js";
+export { isCancelAuctionSupported } from "../../extensions/marketplace/__generated__/IEnglishAuctions/write/cancelAuction.js";
```

Comment:
please follow the pattern we do everywhere else, this should be exported from the same file that contains the actual function, it's fine to re-export it from there if it doesn't need anything else, but keep it next to the implementation so if the implementation changes in the future it's easier to remember to also change the detection logic